keep/api/routes/mapping.py

```diff
@@ -108,6 +111,8 @@ def update_rule(
     existing_rule.matchers = rule.matchers
     existing_rule.file_name = rule.file_name
     existing_rule.priority = rule.priority
+    existing_rule.updated_by = authenticated_entity.email
+    existing_rule.last_updated_at = datetime.datetime.utcnow()
```

Comment:
https://news.ycombinator.com/item?id=38333116
```suggestion
    existing_rule.last_updated_at = datetime.datetime.now(tz=datetime.timezone.utc)
```

Reply:
Wow didn't know that it is deprecated, fixed it.